docs/data/charts/bars/HorizontalBars.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { axisClasses } from '@mui/x-charts';
+
+const chartSetting = {
+  xAxis: [
+    {
+      label: 'rainfall (mm)',
+    },
+  ],
+  width: 500,
+  height: 300,
+  sx: {
+    [`.${axisClasses.left} .${axisClasses.label}`]: {
+      transform: 'rotate(-90deg) translate(0px, -20px)',
+    },
+  },
```

Comment:
Am I missing something or is it actually not needed? 🤔

Reply:
Effectively, there is no label for this axis.
I saved some space by assuming people know that Jan, Fev, ... are months 😁

Reply:
Hm... 
I'm either really blind or just really can't see a difference with or without this prop added 🤷 🙈 🤦

Reply:
Your not blind, I removed the axis label and forgot to remove its customisation 🙈

----

For more detail, adding the label can be done as follow

```diff
-      yAxis={[{ scaleType: 'band', dataKey: 'month' }]}
+      yAxis={[{ scaleType: 'band', dataKey: 'month', label: 'month' }]}
```

And here is the impact of this code

## With
![image](https://github.com/mui/mui-x/assets/45398769/b8918424-ff88-47fe-8506-881431e598cb)

## Without
![image](https://github.com/mui/mui-x/assets/45398769/babb5be8-6b8a-4050-a1ab-bd00fc87f0cb)

Reply:
Thanks for clarifying! 👍